api4/cloud.go

```diff
@@ -51,18 +51,43 @@ func (api *API) InitCloud() {
 	// POST /api/v4/cloud/webhook
 	api.BaseRoutes.Cloud.Handle("/webhook", api.CloudAPIKeyRequired(handleCWSWebhook)).Methods("POST")
 
-	api.BaseRoutes.Cloud.Handle("/notify-admin-to-upgrade", api.APISessionRequired(handleNotifyAdminToUpgrade)).Methods("POST")
+	api.BaseRoutes.Cloud.Handle("/notify-admin", api.APISessionRequired(handleNotifyAdmin)).Methods("POST")
+	api.BaseRoutes.Cloud.Handle("/trigger-notify-admin-posts", api.APISessionRequired(handleTriggerNotifyAdminPosts)).Methods("POST")
```

Comment:
Seems like this trigger endpoint is only needed to allow end to end tests to request the notifications to happen immediately instead of through the job. I see in the handler its locked down to admins, which is good.

Can we condition the adding of this endpoint on whether `config.EnableTesting` is set to true? I'm *guessing* e2e runs have that set to true (spinwicks definitely do; that's why the `/test threaded_post` commands work there.)